Chummer/Backend/Static/Managers/ImprovementManager.cs

```diff
@@ -1343,252 +1343,257 @@ string strLimitToAttribute
                                               XmlNode nodBonus, int intRating = 1, string strFriendlyName = "",
                                               bool blnAddImprovementsToCharacter = true)
         {
-            Log.Debug("CreateImprovements enter");
-            Log.Info("objImprovementSource = " + objImprovementSource);
-            Log.Info("strSourceName = " + strSourceName);
-            Log.Info("nodBonus = " + nodBonus?.OuterXml);
-            Log.Info("intRating = " + intRating.ToString(GlobalSettings.InvariantCultureInfo));
-            Log.Info("strFriendlyName = " + strFriendlyName);
+            string trace = "CreateImprovements enter" + Environment.NewLine;
```

Comment:
Maybe might be better to have this as a StringBuilder, but otherwise I don't see anything wrong.